src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization/ResourceUtilizationHealthCheckExtensions.cs

```diff
@@ -30,7 +29,7 @@ public static IHealthChecksBuilder AddResourceUtilizationHealthCheck(this IHealt
         _ = Throw.IfNull(builder);
         _ = Throw.IfNull(tags);
 
-        _ = builder.Services.AddValidatedOptions<ResourceUtilizationHealthCheckOptions, ResourceUtilizationHealthCheckOptionsValidator>();
+        _ = builder.Services.AddOptionsWithValidateOnStart<ResourceUtilizationHealthCheckOptions, ResourceUtilizationHealthCheckOptionsValidator>();
```

Comment:
Assuming customers will also have to adapt for these sorts of changes, I wonder if this is something that UpgradeAssistant supports as well (meaning mapping static methods, since what we've done so far is mapping NuGet packages and renamed types) cc @RussKie

Reply:
This is not the only API rename :)
My understanding is that the .NET UA supports such conversions.